tests/ProGatewayTest.php

```diff
@@ -20,7 +20,7 @@ public function setUp()
                 'lastName' => 'User',
                 'number' => '4111111111111111',
                 'expiryMonth' => '12',
-                'expiryYear' => '2017',
+                'expiryYear' => '2019',
```

Comment:
I assume this is required to be a date in the future? In that case I'd use `(new \Datetime())->modify('+1 year')->format('Y')` to avoid having to update the test every year

Reply:
```suggestion
                'expiryYear' => (new \Datetime())->modify('+1 year')->format('Y'),
```

Reply:
This also applies to the other test :)